config/routes.rb

```diff
@@ -1,5 +1,12 @@
 Rails.application.routes.draw do
-  root "articles#index"
+  root 'sessions#welcome'
+
+  get 'login', to: 'sessions#new'
```

Comment:
@idprietop why not just use a resources method to cover all the session functions?

Reply:
You're right it's a better idea, I didn't do it because I applied it the same way it was in the documentation.